packages/repository/src/model.ts

```diff
@@ -119,6 +119,15 @@ export class ModelDefinition {
     return this;
   }
 
+  /**
+   * Define a new relation.
+   * @param definition The definition of the new relation.
+   */
+  addRelation(definition: RelationMetadata): this {
+    this.relations[definition.name] = definition;
+    return this;
```

Comment:
I'm curious; why do we return the model definition here? wouldn't the first line suffice?

Reply:
See https://en.wikipedia.org/wiki/Fluent_interface

> A fluent interface is normally implemented by using method chaining to implement method cascading (in languages that do not natively support cascading), concretely by having each method return this (self).